tests/CMakeLists.txt

```diff
@@ -229,6 +229,15 @@ macro(ssg_refcheck_test PRODUCT PROFILE KEY)
     set_tests_properties("refchecker-${PRODUCT}-${PROFILE}" PROPERTIES LABELS quick)
 endmacro()
 
+macro(stig_srg_mapping PRODUCT)
```

Comment:
I think it would be better to add "test" to the name of macro, that will help people who see the invocation of the macro to realize what the macro actually does just by seeing its name.